server/index.js

```diff
@@ -7,7 +7,7 @@ const db = require('../db/index.js');
 app.use(express.static(path.join(__dirname, '/../public/dist')));
 
 app.get('/item', (req, res) => {
-  const itemId = req.query.id || 20;
+  const itemId = req.query.id || Math.floor(Math.random() * 50) || 36;
   db.retrieveListingImages(itemId, (results) => {
     res.json(results);
   });
```

Comment:
If there was an error with the database retrieving images, what would happen here?